packages/odata-service-inquirer/src/index.ts

```diff
@@ -1,12 +1,18 @@
 import { type InquirerAdapter } from '@sap-ux/inquirer-common';
+import type { Question } from 'inquirer';
 import { type Logger } from '@sap-ux/logger';
 import { OdataVersion } from '@sap-ux/odata-service-writer';
 import { type ToolsSuiteTelemetryClient } from '@sap-ux/telemetry';
 import autocomplete from 'inquirer-autocomplete-prompt';
 import { ERROR_TYPE, ErrorHandler, setTelemetryClient } from '@sap-ux/inquirer-common';
 import { initI18nOdataServiceInquirer } from './i18n';
 import { getQuestions } from './prompts';
-import { SystemSelectionAnswerType } from './prompts/datasources/sap-system/system-selection';
+import {
+    type SystemSelectionAnswers,
```

Comment:
Unused import?

Reply:
If you are referring to the import the appears deleted in the diff, it was not deleted. You could see it in line 12.